extensions/common/sql/sql-lease/src/test/java/org/eclipse/edc/sql/lease/TestEntityLeaseStatements.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *       SAP SE - refactoring
+ *
+ */
+
+package org.eclipse.edc.sql.lease;
+
+class TestEntityLeaseStatements implements LeaseStatements {
```

Comment:
this should be moved back as internal class to `PostgresLeaseContextTest`

Reply:
May I know why? The class is already package private and inside a test package. It has no visibility elsewhere.

Reply:
unless it needed to be used by some other tests I would let it as an internal class.